### ISSUE TITLE
No option to create unselectable overlays

### ISSUE THREAD

**Issue by localjo** [openlayers/openlayers#7241]
Currently, [all overlays get the class `ol-selectable`](https://github.com/openlayers/openlayers/blob/master/src/ol/overlay.js#L62), regardless of their purpose. In our app, none of the overlays we're using should be selectable by the user, and I imagine that this is a common use case. Writing CSS to override the existing styles and make `.ol-selectable` unselectable is both counter intuitive and makes it difficult to have some overlays which are selectable, while others are not. I propose adding an option to give overlays the `ol-unselectable` class instead. Example;
```
new ol.Overlay({
  element: overlayEl,
  id: overlayId,
  selectable: false
});
```

Happy to open a PR if someone can confirm that my understanding is correct and that I'm not simply misunderstanding how to control an overlay's selectability.

**Comment by fredj** [comment]
Note: the selectable css class was added in #6741

**Comment by fredj** [comment]
Instead of having a `selectable` option, what about adding a `className` like all the ol.controls?
The default value will be `ol-overlay-container ol-selectable`.

**Comment by localjo** [comment]
Yes, I think being consistent with other ol.controls is a good idea. A class name option should work just as well.

**Comment by fredj** [comment]
A PR will be greatly appreciated!

**Comment by bobobobo** [comment]
PR #7377 implements support for className on ol.Overlay

**Comment by fredj** [comment]
fixed with #7377
